gc/base/GCExtensionsBase.hpp

```diff
@@ -1105,31 +1105,94 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 
 	MMINLINE MM_Heap* getHeap() { return heap; }
 
-#if defined(OMR_GC_MODRON_SCAVENGER)
 	MMINLINE void
 	setGuaranteedNurseryRange(void* start, void* end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		_guaranteedNurseryStart = start;
 		_guaranteedNurseryEnd = end;
+#endif /* OMR_GC_MODRON_SCAVENGER */
 	}
 
 	MMINLINE void
 	getGuaranteedNurseryRange(void** start, void** end)
 	{
+#if defined(OMR_GC_MODRON_SCAVENGER)
 		*start = _guaranteedNurseryStart;
 		*end = _guaranteedNurseryEnd;
+#else /* OMR_GC_MODRON_SCAVENGER */
+		*start = NULL;
+		*end = NULL;
+#endif /* OMR_GC_MODRON_SCAVENGER */
+	}
+
+	MMINLINE bool isRememberedSetInOverflowState()
```

Comment:
To match existing formatting in this file. Function name is on a new line. Applies to all other instances.
```suggestion
	MMINLINE bool
	isRememberedSetInOverflowState()
```